include/util/json.hpp

```diff
@@ -20186,7 +20186,7 @@ class basic_json
     @note Floating-point inside JSON values numbers are compared with
     `json::number_float_t::operator==` which is `double::operator==` by
     default. To compare floating-point while respecting an epsilon, an alternative
-    [comparison function](https://github.com/mariokonrad/marnav/blob/master/src/marnav/math/floatingpoint.hpp#L34-#L39)
```

Comment:
Looks like it should be https://github.com/mariokonrad/marnav/blob/01c55205736fcc8157891b84e3efe387a221ff3a/include/marnav/math/floatingpoint.hpp#L33-#L38 (note the line numbers)

Reply:
Sounds great! It has been fixed.